### DISCUSSION TITLE
OnnxRuntime with AMD E9171 gpu

### DISCUSSION THREAD

**Discussion by drvshavva** [microsoft/onnxruntime#9591] [Other Q&A]
hi, can i use onnxruntime for inference with AMD E9171 gpu? Thanks in advance for your answers

**Comment by snnn** [comment]
https://github.com/RadeonOpenCompute/ROCm#Hardware-and-Software-Support

It seems not.

**Reply by drvshavva** [reply]
Thank you for your answer

**Reply by drvshavva** [reply]
according to this page this gpu support rocm:

https://www.techpowerup.com/gpu-specs/amd-lexa.g806
